lib/utils/Constants.ts

```diff
@@ -66,3 +66,7 @@ export const NBO_ERC20_CONTRACT = '0x6BCa3B77C1909Ce1a4Ba1A20d1103bDe8d222E48';
 export const DAO_MULTISIG = '0x65a5076c0ba74e5f3e069995dc3dab9d197d995c';
 export const STAKING_CONTRACT_V1 = '0x25d28a24Ceb6F81015bB0b2007D795ACAc411b4d';
 export const BONDING_CALCULATOR = '0x0b8d6D6611Ed7cCe01BbcC57826548C6107B0478';
+
+//Vesting constants
+export const C3_WSKLIMA_CONTRACT = "0xe02efadA566Af74c92b6659d03BAaCb4c06Cc856"
+export const C3_WSKLIMA_TOKEN = "WSKLIMA"
```

Comment:
This should be named something more specific like "C3-wsKLIMA" so we can differentiate if more wsKLIMA staking gauges come down the line